frappe/email/__init__.py

```diff
@@ -26,6 +29,8 @@ def get_users():
 		)
 		if not out:
 			out = get_users()
+		if not out:
```

Comment:
lets get only from contact since Contact is part of Frappé now.

Might also be a good idea to create a Contact for every User (if missing).